<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCheckBox" name="LED_1_chk">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>85</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>LED 1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="LED_2_chk">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>85</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>LED 2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="LED_3_chk">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>85</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>LED 3</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="LED_4_chk">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>85</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>LED 4</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Beep_chk">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>85</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Beep</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Mode_1_btn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>模式一</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Mode_2_btn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>模式二</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Mode_3_btn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>模式三</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Mode_4_btn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>模式四</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Clear_btn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
